sdk/easm/azure-analytics-defender-easm/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo" : "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath" : "java",
+  "TagPrefix" : "java/easm/azure-analytics-defender-easm",
+  "Tag" : ""
```

Comment:
Have you run the recording for the tests? The tag should not be empty.

Reply:
@haolingdong-msft, I was able to run tests in both RECORD and PLAYBACK mode successfully. However, the assets.json file isn't updated. Is there any steps that I missed here?

Reply:
assets.json should be updated automatically by the testproxy when running in RECORD mode. Could you show us your  running command？

Reply:
Add @samvaity  for testproxy related issue as well.